db/dbErrors.js

```diff
@@ -71,6 +71,25 @@ dbErrors.create({
   explanation: 'If a subject specifies that it has a parent, the parent ' +
    'subject must already exist.',
 });
+dbErrors.create({
+  code: 10115,
+  status: 400,
+  name: 'ParentSubjectNotMatch',
+  parent: dbErrors.ValidationError,
+  range: [],
```

Comment:
I included it due to it being in the previous error. I will take it out as it doesn't seem necessary.